wdl-lint/tests/lints/one-line-after-version/source.errors

```diff
@@ -4,11 +4,3 @@ error: there must be at least one task, workflow, or struct definition in the fi
 6 │ 
   │  
 
-note[VersionFormatting]: expected exactly one blank line after the version statement
-  ┌─ tests/lints/one-line-after-version/source.wdl:5:12
-  │  
-5 │   version 1.1
```

Comment:
Is this right?

Reply:
Maybe? I changed the code so that `VersionFormatting` won't emit anything if it's the last thing (aside from whitespace) in the document. That case is a clear `error` (which is already being handled), and the lints emitted at the same time seem inconsequential to me 🤷